tests/parser/SimulationConfigTest.cpp

```diff
@@ -106,6 +106,15 @@ const std::string& inputStr_vap_dis = "RUNSPEC\n"
                                       "REGIONS\n"
                                       "\n";
 
+const std::string& inputStr_watvap = "RUNSPEC\n"
```

Comment:
Just a tip for the next time - for all new test strings of this type we try to use multiline string literals:
```C++
const std::string inputStr = R"(
RUNSPEC
VAPWAT
DIMENS
 10 3 4 /
GRID
REGIONS
)";
```
then you do not have to fiddle so much with `"` and `\n`.

Reply:
Thank you for the tip.